extensions/amp-lightbox/1.0/amp-lightbox.css

```diff
@@ -18,4 +18,18 @@ amp-lightbox {
   width: 100%;
   height: 100%;
   position: fixed;
+  visibility: hidden;
```

Comment:
Let's just a comment explaining why we have `visibility:hidden` - e.g. it's canceled by the lightbox itself.

Reply:
Done